pilot/docker/Dockerfile.proxyv2

```diff
@@ -24,21 +19,7 @@ ADD envoy_policy.yaml.tmpl /etc/istio/proxy/envoy_policy.yaml.tmpl
 ADD envoy_telemetry.yaml.tmpl /etc/istio/proxy/envoy_telemetry.yaml.tmpl
 ADD istio-iptables.sh /usr/local/bin/istio-iptables.sh
```

Comment:
what precisely does istio-iptables.sh do?  If its a bash script, does it not need bash to operate?  Is it used in the implementation?